code/modules/organs/subtypes/machine.dm

```diff
@@ -291,6 +291,25 @@
 	else
 		..()
 
+/obj/item/organ/internal/ipc_tag/proc/modify_tag_data()
+	if(!owner || owner.stat)
+		return
+	var/new_citizenship = tgui_input_list(owner, "Choose a citizenship for your IPC tag.", "Tag Citizenship", CITIZENSHIPS_ALL_IPC)
+	if(!new_citizenship)
+		return
+	citizenship_info = new_citizenship
+	var/new_ownership = tgui_input_list(owner, "Choose an ownership status for your IPC tag.", "Tag Ownership", list(IPC_OWNERSHIP_COMPANY, IPC_OWNERSHIP_PRIVATE, IPC_OWNERSHIP_SELF))
```

Comment:
I recommend you swap it. Ownership first, if it isnt ownership_self, skip citizenship